### PR TITLE
issue #13497 - Method getUrl in Magento\Catalog\Model\Product\Attribu…

### PR DESCRIPTION
**Pull request by igortregub**
### Preconditions

1. Magento 2.2.2
2. 

### Steps to reproduce

1. Try to get product image via method getUrl from class Magento\Catalog\Model\Product\Attribute\Frontend

### Fixed Issues
1. https://github.com/magento/magento2/issues/13497 Method getUrl in Magento\Catalog\Model\Product\Attribute\Frontend returns image url with double slash

### Expected result

1. returns image url without double slash 

### Actual result

1. will be returned image with double slash
screen shot 2018-02-05 at 12 37 04 pm 2018-02-05 13-25-55

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=13498) <br/>All committers have signed the CLA.

File: app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php [hunk 1]
```diff
@@ -43,8 +48,8 @@ public function getUrl($product)
         $url = false;
         if (!empty($image)) {
             $url = $this->_storeManager->getStore($product->getStore())
-                ->getBaseUrl(\Magento\Framework\UrlInterface::URL_TYPE_MEDIA)
-                . 'catalog/product/' . $image;
+                    ->getBaseUrl(UrlInterface::URL_TYPE_MEDIA)
+                . 'catalog/product' . $image;
```
Comment (orlangur) [inline app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php]: As `$image` may come from various sources there is no guarantee it contains leading slash.

Please 

1. change logic to `. 'catalog/product/' . ltrim($image, '/')` instead 
1. create a separate test case to assure there is no slash duplication
1. as always, do an amend commit and force push

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php (hunk 1 above)
Comment (igortregub) [reply app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php]: ok, thank you

**Review by ihor-sviziev — APPROVED** [review]

File: app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php (hunk 1 above)
Comment (orlangur) [reply app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php]: @igortregub changes look good now, please squash changes into single commit and force push.

Comment (igortregub) [reply app/code/Magento/Catalog/Model/Product/Attribute/Frontend/Image.php]: @orlangur, done

**Comment by magento-engcom-team** [comment]
@igortregub thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Review by orlangur — APPROVED** [review]

**Pull request merged**